src/test/resources/.gitignore

```diff
@@ -0,0 +1,3 @@
+# We blanket-ignore .bam files in the top-level of the repo, but want to undo that here.
+!*.bam
```

Comment:
Yea this caused me some problems when I tried to `git mv adam-core/* ./` to put all of adam-core in the top directory (after `git rm`ing the other modules); `sorted.bam*` did not come with that move since they were `.gitignore`d (despite having been `git add`ed to the repo at some point, maybe before they became `.gitignore`d?), so I was just missing them in my builds, which took me a while to figure out.

In any case, yea `.gitignore` "whitelists" can be useful in addition to the way it's usually used.